tools/update-client

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# This script creates a new release of the Hypothesis browser extension
+# using the latest version of the client available on npm.
+
+set -eu
+
+LATEST_CLIENT_VERSION=$(npm view hypothesis version)
+
+# Update checkout
+git checkout master
+git fetch
+git reset --hard origin/master
```

Comment:
I think this could delete uncommitted changes. Do you think it should run something like `git status --short` before it does any of these git commands? (And the script should crash with an error if there is any output from `git status --short`)

We could also make it do `git branch | grep '^\*'` and if the output isn't `* master` then abort, rather than changing the user's branch on them (potentially without them noticing)